src/elements/Input.js

```diff
@@ -6,14 +6,17 @@ export function StyledInput({
   className,
   classNameWrapper = "w-full",
   error = null,
+  size = null,
   ...props
 }) {
   return (
     <div
       className={`flex flex-col relative items-start text-black mb-[10px] ${classNameWrapper}`}
     >
       <input
-        className={`text-white px-5 w-full h-[60px] text-14 appearance-none focus:outline-none bg-greyDark placeholder:text-grey ${className}`}
+        className={`text-white w-full text-14 appearance-none focus:outline-none bg-greyDark placeholder:text-grey ${
+          size ? "h-[30px] px-4" : "h-[60px] px-5"
```

Comment:
Better have an object to group styles by some specific name
For example : 
`{
    small : 'h-[30px] px-4',
    default: 'h-[60px] px-5',
  }`
  And then pass corresponding prop to the element. You can set `size` to have a default value ,`default` for example. 
  Going this way we'll have more scalable solution.